docs/tutorials/resolution_functions.ipynb

```diff
@@ -0,0 +1,380 @@
+    "from EasyReflectometry.sample import Material\n",
+    "from EasyReflectometry.sample import Multilayer\n",
+    "from EasyReflectometry.sample import Sample\n",
+    "from EasyReflectometry.plot import plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fd3e8f7-84ac-41c4-a89d-922ed82a001e",
+   "metadata": {},
+   "source": [
+    "For reference we fetch the version of the software packages we are using. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "549734c1-bbd9-41f3-8a20-d7a8ded37802",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'numpy: {np.__version__}')\n",
+    "print(f'scipp: {sc.__version__}')\n",
+    "print(f'EasyReflectometry: {EasyReflectometry.__version__}')\n",
+    "print(f'Refl1D: {refnx.__version__}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "687719c7-d5a2-4d60-953f-c854c5ca1c4a",
+   "metadata": {},
+   "source": [
+    "## Reading in measured data\n",
+    "\n",
+    "The data that we will investigate in this tutorial was generated with Refnx and are stored in `.ort` [format file](https://github.com/reflectivity/file_format/blob/master/specification.md) files. In this tutorial we are investigation how we can include resolution effects when simulating and reproducing data measured in an experiment.  For an `.ort` file the resoultion data for reflectivity is stored in the fourth column.\n",
+    "\n",
+    "IMPORTANT when using `EasyReflectometry` functionality for loading an `.ort` file we store the resolution data as a variance (squared value).  As a consequence one needs to take the squareroot of the loaded data to recover the raw values (fourth column)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "609174e5-1371-412d-a29f-cb05bfe36df0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dict_reference = {}\n",
+    "dict_reference['0'] = load(\"_static/resolution/mod_pointwise_two_layer_sample_dq-0.0.ort\")\n",
+    "dict_reference['1'] = load(\"_static/resolution/mod_pointwise_two_layer_sample_dq-1.0.ort\")\n",
+    "dict_reference['10'] = load('_static/resolution/mod_pointwise_two_layer_sample_dq-10.0.ort')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ab3a164-62c8-4bd3-b0d8-e6f22c83dc74",
+   "metadata": {},
+   "source": [
+    "As an example we can plot the reference data without any resolution effects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "31ab44d3-826a-4270-9046-ad667dcb66ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot(dict_reference['0'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dad78ccc-1e6f-47cd-8557-c4fa6c736f4b",
+   "metadata": {},
+   "source": [
+    "## Building our model\n",
+    "\n",
+    "The system that was used to produce the data shown above is based on a silicon subphase, with a repeating multilayer of nickel and titanium grown upon it. \n",
```

Comment:
The system is not nickel or titanium, it is arbitrary SLD's. We can say it is two metal layers (without being specific), whose SLD's are 4 and 8. It is also not a repeating multilayer, just one layer of each, unless you would describe it being a repeat with zero repeats.

Reply:
Thanks, 
whoops, some copy - paste left behind.
Fixed